packages/nimble-components/src/anchor-menu-item/specs/README.md

```diff
@@ -0,0 +1,145 @@
+# Nimble Anchor Menu Item
+
+## Overview
+
+The `nimble-anchor-menu-item` is a link that can be used within a `nimble-menu` and is styled to look just like a `nimble-menu-item`.
+
+### Background
+
+[Visual Design spec (menu item)](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/c098395e-30f8-4bd4-b8c5-394326b59919)
+[GitHub issue](https://github.com/ni/nimble/issues/1020)
+
+### Non-goals
+
+-   Support for checked state (checkbox or radio button)
+-   Support for nesting, expanding/collapsing
+
+### Features
+
+-   Supports icon
+-   Disabled state
+-   Follows menu item navigation pattern
+
+### Risks and Challenges
+
+None
+
+### Prior Art/Examples
```

Comment:
Do we have known use cases for this component in SLE or other clients? I think the answer is yes. Not a huge deal whether or not you document them here but at some point we should figure out what the integration story will be once this component is done.

Let me know if you want to track down use cases and capture them somewhere or if you'd like the Fred-Jesse PO conglomerate to take it on

Reply:
It's used in the help menu within SLE...or, at least it used to be. I don't even see the help menu on any apps right now.

Reply:
I don't immediately know how to find instances of it, but I'll give it a try and ask for help if I need it.